content/platforms/kubernetes/quick-start.md

```diff
@@ -0,0 +1,164 @@
+described in the previous section.
+
+You can install the operator with a single apply command:
+
+```
+kubectl apply -f bundle.yaml -n my-namespace
+```
+
+You should see a result similar to:
+
+```
+role.rbac.authorization.k8s.io/redis-enterprise-operator created
+serviceaccount/redis-enterprise-operator created
+rolebinding.rbac.authorization.k8s.io/redis-enterprise-operator created
+customresourcedefinition.apiextensions.k8s.io/redisenterpriseclusters.app.redislabs.com configured
+customresourcedefinition.apiextensions.k8s.io/redisenterprisedatabases.app.redislabs.com configured
+deployment.apps/redis-enterprise-operator created
+```
+
+## Verifying the operator is running
+
+You can verify the operator is running in your namespace checking the deployment:
+
+```
+kubectl get deployment -l name=redis-enterprise-operator -n my-namespace
+```
+
+You should see a result similar to:
+
+```
+NAME                        READY   UP-TO-DATE   AVAILABLE   AGE
+redis-enterprise-operator   1/1     1            1           0m36s
+```
+
+## Testing the operator
+
+A cluster is created by creating a custom resource with the kind "RedisEnterpriseCluster"
+that contains the specification of the cluster option. See the
+[Redis Enterprise Cluster API reference](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md)
+for more information on the various options available.
+
+You can test the operator by creating a minimal cluster by following this procedure:
+
+1. Create a file called `simple-cluster.yaml` that defines a Redis Enterprise
+   cluster with three nodes:
+
+    ```
+    cat <<EOF > simple-cluster.yaml
+    apiVersion: "app.redislabs.com/v1"
+    kind: "RedisEnterpriseCluster"
+    metadata:
+      name: "test-cluster"
+    spec:
+      nodes: 3
+    EOF
+    ```
+2. Create the CRD in the namespace with the file `simple-cluster.yaml`:
+
+    ```
+    kubectl apply -f simple-cluster.yaml -n my-namespace
+    ```
+
+    You should see a result similar to:
+
+    ```
+    redisenterprisecluster.app.redislabs.com/test-cluster created
+    ```
+
+3. You can verify the creation of the with:
+
+    ```
+    kubectl get rec -n my-namespace
+    ```
+
+    You should see a result similar to:
+
+    ```
+    NAME           AGE
+    test-cluster   1m
+    ```
+
+
+   At this point the operator will go through the process of creating various
+   services and pod deployments. You can track the progress by examining the
+   StatefulSet associated with the cluster:
+
+   ```
+   kubectl rollout status sts/test-cluster -n my-namespace
+   ```
+
+   or by simply looking at the status of all the resources in your namespace:
+
+   ```
+   kubectl get all -n my-namespace
+   ```
+
+4. Once the cluster is running, follow the information in [Managing Redis Enterprise Databases in Kubernetes]({{< relref "db-controller.md" >}})
+   to create databases.
```

Comment:
why not add this here and make it a "to a working database" quick-start? what most people want is a working endpoint(?)

Reply:
Good point - added.